.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx --no -- commitlint --edit ${1}
```

Comment:
Very cool! This assumes we have npx installed though. Let's add this to the documentation or add it to dependencies.

Reply:
npx comes with npm which comes with node so we shouldn't need to document npx specifically

Reply:
A sweet. It’s bundled with npm since 7.0.0